core/flyout.js

```diff
@@ -427,9 +581,14 @@ Blockly.Flyout.prototype.show = function(xmlList) {
   this.listeners_.push(Blockly.bindEvent_(this.svgBackground_, 'mouseover',
       this, deselectAll));
 
-  this.width_ = 0;
+  if (this.horizontalLayout_) {
+    this.height_ = 0;
+  } else {
+    this.width_ = 0;
+  }
   this.reflow();
 
+  this.offsetHorizontalRtlBlocks(this.workspace_.getTopBlocks(false));
```

Comment:
why do we need this if there is no movement on mouse wheel in the horizontal case?